test/1_test_fullpath.ts

```diff
@@ -1333,16 +1337,24 @@ describe('Voucher tests', () => {
       await contractMockBosonRouter.deployed();
 
       //Set mock so that failed transferFrom of tokens with no return value can be tested in transferFromAndAddEscrow
+      await contractBosonRouter.pause();
+
       await contractCashier.setBosonRouterAddress(
         contractMockBosonRouter.address
       );
 
+      await contractVoucherKernel.setBosonRouterAddress(
+        contractMockBosonRouter.address
+      );
+
+      await contractMockBosonRouter.pause();
+      await contractMockBosonRouter.unpause();
```

Comment:
Why does this test pause then immediately unpause the mock boson router without doing anything inbetween? Add a comment if this isn't an oopsy.

Reply:
It's needed in the current implementation. I added a comment explaining why.